src/scripts.js

```diff
@@ -1,43 +1,54 @@
 import './css/base.scss';
 import './css/style.scss';
-
 import './images/person walking on path.jpg';
 import './images/The Rock.jpg';
-
-import userData from './data/users';
-import hydrationData from './data/hydration';
-import sleepData from './data/sleep';
-import activityData from './data/activity';
-
 import User from './User';
 import Activity from './Activity';
 import Hydration from './Hydration';
 import Sleep from './Sleep';
 import UserRepo from './User-repo';
+import fetchData from './fetchAllData';
+
+const data = {
+  userData: null,
+  sleepData: null,
+  activityData: null,
+  hydrationData: null
+}
 
 let userRepo;
 let hydrationRepo;
 let sleepRepo;
 let activityRepo;
 let currentUser;
 
-window.onload = startApp(); //Starts Here (global variables currentInformation = {})
+window.onload = startApp();
 
 function startApp() {
-  instantiatePageData();
-  populatePage();
+  fetchData() 
+    .then(allData => {
+      data.userData = allData.userData;
+      data.sleepData = allData.sleepData;
+      data.activityData = allData.activityData;
+      data.hydrationData = allData.hydrationData;
```

Comment:
That fetchAllData function is returning the data sets so we have to 1) make is a synchronous process and 2) reassign are data files to whatever's been fetched?